docs/modules/ROOT/pages/migration.adoc

```diff
@@ -479,6 +479,13 @@ class WebSocketSecurityConfig: WebSocketMessageBrokerConfigurer {
 ----
 ====
 
+[[servlet-opt-in-sha256-rememberme]]
+==== Use SHA-256 in Remember Me
+
+The `TokenBasedRememberMeServices` implementation now supports SHA-256 for the Remember Me token and this is the default in Spring Security 6.
```

Comment:
I think it would be helpful to explain why this change was made. Moving from MD5 to SHA-256 is obvious to you and I, but not everyone lives in the security space on a day-to-day basis. I'd strongly recommend at least explaining that MD5 is an insecure default.